src/nncf/common/tensor_statistics/aggregator.py

```diff
@@ -118,6 +118,7 @@ def _load_statistics(self, data: dict[str, Any]) -> None:
         """
         for _, statistic_point, tensor_collector in self.statistic_points.get_tensor_collectors():
             statistics = tensor_collector.get_statistics()
+            statistics = cast(TensorStatistic, statistics)
```

Comment:
Can `get_statistics` return dict[str, any]? 
If it's correct, so there is need to use check `isinstance(statistics, TensorStatistic)` instead of cast